ddl/generated_column.go

```diff
@@ -247,6 +247,9 @@ func (c *illegalFunctionChecker) Enter(inNode ast.Node) (outNode ast.Node, skipC
 		// Aggregate function is not allowed
 		c.hasAggFunc = true
 		return inNode, true
+	case *ast.RowExpr:
+		c.hasIllegalFunc = true
```

Comment:
Do we have a test for this?

Reply:
In mysql_test, it's not in this PR, but I will add it later.